extensions/src/ACRE2Core/FilterRadio.cpp

```diff
@@ -89,6 +94,7 @@ CFilterRadio::CFilterRadio(void)
     srand((unsigned int)time(0));
     this->m_HighPass.setup (TS_SAMPLE_RATE, 750, 0.97);
     this->m_LowPass.setup (TS_SAMPLE_RATE, 4000, 2.0);
+    this->m_lowShelf.setup(TS_SAMPLE_RATE, 1000, -20, 1);
```

Comment:
```suggestion
    this->m_lowShelf.setup(TS_SAMPLE_RATE, 1000, -10, 1);
```
I think 10db will be enough to allow listeners to tell that it's a speaker